tests/test_binary.nim

```diff
@@ -367,7 +367,8 @@ proc testsForSync(binaryPath: static string) =
       +    "example": []
       +  "blurb": "Diffie-Hellman key exchange.",
     """.unindent()
-    let configPath = joinPath("exercises", "practice", "diffie-hellman", ".meta", "config.json")
+    let configPath = joinPath("exercises", "practice", "diffie-hellman",
+        ".meta", "config.json")
```

Comment:
This was auto-formatted by my Nim plugin in VS Code. I can revert, but it will do this every single time

Reply:
Let me double check!

Reply:
I am using that extension